check_times.py

```diff
@@ -0,0 +1,89 @@
+"""Check the frequency of the rebuild loop.
+
+This must be run in a directory that has the ``docsbuild.log.*`` files.
+For example:
+
+.. code-block:: bash
+
+   $ cd docsbuild-logs
+   $ scp "adam@docs.nyc1.psf.io:/var/log/docsbuild/docsbuild.log*" .
+   $ python ../check_times.py
```

Comment:
Something like this?
```suggestion
   $ scp "adam@docs.nyc1.psf.io:/var/log/docsbuild/docsbuild.log*" docsbuild-logs
   $ python check_times.py
```